subjects/crossword/README.md

```diff
@@ -0,0 +1,39 @@
+## crossword
+
+### Instructions
+
+Create the function `crosswordSolver` that is able to solve an empty crossword puzzle.
```

Comment:
What would you think about specifying that the file needs to have that name as well?